compass/ocean/tests/global_ocean/init/remap_ice_shelf_melt.py

```diff
@@ -223,8 +223,8 @@ def remap_paolo(in_filename, base_mesh_filename, culled_mesh_filename,
     ds[field] = area_ratio * fwf
     ds[field].attrs['units'] = 'kg m^-2 s^-1'
     field = 'dataLandIceHeatFlux'
-    ds[field] = (latent_heat_of_fusion *
-                 ds.dataLandIceFreshwaterFlux)
+    ds[field] = -(latent_heat_of_fusion *
+                  ds.dataLandIceFreshwaterFlux)
```

Comment:
@darincomeau and @cbegeman, could you just review this by inspection when you can?

I will post output later this evening.

Reply:
@xylar Yes, I think this is exactly what we want. this is probably obvious, but when you do run the simulations, make sure the conservation AM is on.